src/olympia/devhub/helpers.py

```diff
@@ -115,7 +115,17 @@ def docs_breadcrumbs(context, items=None):
 @jinja2.contextfunction
 def add_file_modal(context, title, action, action_label, modal_type='file'):
     addon = context['addon']
-    upload_url = reverse('devhub.upload_for_addon', args=[addon.slug])
+    version = context.get('version',
+                          addon.find_latest_version_including_rejected())
```

Comment:
The way it pulls `version` automatically from the context instead of passing it to the function when calling it. *Especially* when there is a fallback to getting it from the `addon` instead. It's just hard to debug where the version comes from